src/fides/api/ops/service/connectors/limiter/rate_limiter.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import time
+from enum import Enum
+
+from fides.api.ops.common_exceptions import RedisConnectionError
+from fides.api.ops.util.cache import FidesopsRedis, get_cache
+
+logger = logging.getLogger(__name__)
+
+
+class RateLimiterPeriod(Enum):
+    """
+    Defines the periods which rate limiter supports for time buckets
+    """
+
+    SECONDS = ("seconds", 1)
```

Comment:
nit: the plural's here confused me a bit. effectively this is "n requests per [second/minute/hour/day]", so it sort of seems to me like it should just be singular? but maybe i'm thinking about it incorrectly. not a big deal either way.

Reply:
sure let's change it